odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/DungeonWaypoints.kt

```diff
@@ -71,6 +71,7 @@ object DungeonWaypoints : Module(
     fun onRender(event: RenderWorldLastEvent) {
         DungeonUtils.currentRoom?.waypoints?.forEach {
             Renderer.drawBox(it.toAABB(it.size), it.color, fillAlpha = if (it.filled) .8 else 0, depth = it.depth)
+            Renderer.drawStringInWorld(it.title.toString(), Vec3(it.x+0.5,it.y+0.5,it.z+0.5))
```

Comment:
Replace ´it.title.toString()" with "it.title ?: ""   ´
This removes the need for the function to replace "null"